docs/about.md

```diff
@@ -6,39 +6,45 @@
-* [**Tejas Bhuwania**](https://www.github.com/Tejas2805): _Contributor_ since Jan 2020
-
-Past Members:
-* [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Project Member_ for Jan 2018 - Aug 2018
-* [**Jia Zhixin**](https://github.com/nusjzx): _Project Member_ for May 2018 - Aug 2018
+
+**Dev Team**:
+* [**Ang Ze Yu**](https://www.github.com/ang-zeyu): **_Project Lead_** since Jan 2021, Member since Jan 2020,
+* [**Jonah Tan Jun Zi**](https://www.github.com/jonahtanjz): _Senior Dev_ since Apr 2020, _Member_ since Jan 2020
+* [**Koh Rayson**](https://www.github.com/raysonkoh): _Senior Dev_ since Apr 2020, _Member_ since Jan 2020
+* [**Ong Wei Xiang**](https://www.github.com/wxwxwxwx9): _Senior Dev_ since Apr 2020, _Member_ since Jan 2020
+* [**Ryo Chandra Putra Aramanda**](https://www.github.com/ryoarmanda): _Senior Dev_ since Apr 2020, _Member_ since Jan 2020
+
+**Contributors:**
+* [**Kimberly Ong Ong Hui Qin**](https://www.github.com/kimberlyohq): _Contributor_ since Jun 2021
+
+**%%Past Members:%%**
+* [**Aaron Chong Jun Hao**](https://github.com/acjh): _Project Mentor_ for Jul 2018 - Dec 2020, _Team Lead_ for Aug 2017 - Jun 2018
+* [**Ahmed Bahajjaj**](https://www.github.com/madanalogy): _Contributor_ for Jan-April 2020
+* [**Chng Zhi Xuan**](https://github.com/Chng-Zhi-Xuan): _Member_ for May 2018 - Jul 2019
+* [**Chua Yun Zhi Nicholas**](https://github.com/nicholaschuayunzhi): _Member_ for Jan 2018 - May 2019
+* [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Member_ for Jan 2018 - Aug 2018
+* [**Daryl Chan**](https://www.github.com/dvrylc): _Contributor_ for Jan-April 2020
+* [**Daryl Tan**](https://www.github.com/openorclose): _Member_ for May 2019 - May 2020
+* [**Jamos Tay**](https://github.com/jamos-tay): _Member_ for Aug 2018 - Apr 2019
+* [**Jeremy Tan Jie Rui**](https://www.github.com/Parcly-Taxel): _Contributor_ for Jan-April 2020
+* [**Jia Zhixin**](https://github.com/nusjzx): _Member_ for May 2018 - Aug 2018
 * [**Jiang Sheng**](https://github.com/Gisonrg): _Founding Member_ and _Team Lead_ for Aug 2016 - Jul 2017
-* [**Lin Si Jie**](https://www.github.com/sijie123): _Project Member_ for Jan 2019 - Aug 2019
-* [**Lu Yang Kenneth**](https://github.com/luyangkenneth): _Project Member_ for Jan 2019 - Aug 2019
-* [**Maddi Aadyaa**](https://www.github.com/amad-person): _Project Member_ for Jan 2019 - Aug 2019
-* [**Rachael Sim Hwee Ling**](https://github.com/rachx): _Project Member_ for Jan 2018 - Dec 2018
-* [**Tan Zhen Yong**](https://www.github.com/Xenonym): _Project Member_ for Jan 2019 - Aug 2019
+* [**Joshua Wong**](https://www.github.com/hcwong): _Contributor_ for Jan-April 2020
+* [**Labayna Neil Brian Narido**](https://www.github.com/nbriannl): _Member_ for Jan - Apr 2020
+* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Mentor_ for Jan - Apr 2020, _Member_ for Jan 2019 - Aug 2019
```

Comment:
Think this should be 

```
* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Lead_ for Jan - Aug 2020, _Member_ for Jan 2019 - Aug 2019
```
🤔  

Wonder if we should switch to one of these for non-lead roles as well, might miss something out here
https://github.com/all-contributors/all-contributors

Reply:
> Wonder if we should switch to one of these for non-lead roles as well, might miss something out here

after this pr*

Reply:
*I'm not exactly too sure how these work or if they are even automatable though 😅 (I'm assuming they are)

Reply:
We can certainly give it a try. 👍

Reply:
👍 

up https://github.com/MarkBind/markbind/issues/1646